InfiniteRecharge/src/main/java/frc/robot/config/Config.java

```diff
@@ -32,7 +35,7 @@
     public int BALLMANAGEMENT_MOTOR_ID = 14;
 
     // Intake
-    public int INTAKE_MOTOR_ID = 10;
+    public static final int INTAKE_MOTOR_ID = 10;

```

Comment:
None of the other Ids are static finals, but maybe they should be. Our Id inheritance is weird.